src/jdk.jfr/share/classes/jdk/jfr/internal/Options.java

```diff
@@ -48,7 +48,7 @@
     private static final int DEFAULT_STACK_DEPTH = 64;
     private static final boolean DEFAULT_SAMPLE_THREADS = true;
     private static final long DEFAULT_MAX_CHUNK_SIZE = 12 * 1024 * 1024;
-    private static final SafePath DEFAULT_DUMP_PATH = SecuritySupport.USER_HOME;
+    private static final SafePath DEFAULT_DUMP_PATH = new SafePath(".");
```

Comment:
Why do need to change DEFAULT_DUMP_PATH here?

Reply:
Emergency dump will be dumped on current work directory now. This default value does not effect. So I changed it to current behavior, and it is nature because other dumps (hs_err, cireplay) are also dumped on current directory.

Reply:
Make sense, I think we could remove `SecuritySupport.USER_HOME` since no place uses it now.

Reply:
I removed `SecuritySupport.USER_HOME` in new commit.